index.d.ts

```diff
@@ -0,0 +1,223 @@
+// Type definitions for ewelink-api
+// Definitions by: Alexander Méhes https://github.com/BMXsanko
+
+declare module 'ewelink-api' {
+  export default eWelink;
+
+    class eWelink {
+    constructor({ }: { email: string; password: string; } | { at: string; } | { at: string; apiKey: string; });
+    /**
+    * Login into eWeLink API and get auth credentials.
+    */
+    login(): Promise<LoginInfo>
+    /**
+    * Opens a socket connection to eWeLink and listen for real-time events.
+    */
+    openWebSocket(callback: (data: {}) => void): Promise<any>
+    /**
+     * Returns a list of devices associated to logged account.
+     */
+    getDevices(): Promise<Device[]>
+    /**
+     * Return information for specified device.
+     */
+    getDevice(deviceId: string): Promise<Device>
+    /**
+     * Query for specified device power status.
+     */
+    getDevicePowerState(deviceId: string, channel?: number): Promise<DeviceState>
+    /**
+     * Change specified device power state.
+     */
+    setDevicePowerState(deviceId: string, channel?: number): Promise<DeviceState>
```

Comment:
This declaration isn't correct. It should be:
```suggestion
    setDevicePowerState(deviceId: string, state?: string, channel?: number): Promise<DeviceState>
```

Reply:
thanks @ducpmm